lara-typescript/src/page-item-authoring/common/components/interactive-iframe.tsx

```diff
@@ -97,9 +97,9 @@ export const InteractiveIframe: React.FC<Props> = (props) => {
   }, [src, resetCount]);
 
   let computedHeight = 300;
-  if(height !== null) {
+  if (height !== null) {
```

Comment:
looks like we need to have travis run the linter on lara-typescript so I can't get away with this in the future. :)